README.md

```diff
@@ -230,6 +233,13 @@ some cases drastically so.
 
 See docs/gomod.md for more details.
 
+### [maven][]
+
+Hermeto supports maven by parsing one or more `lockfile.json` files generated by the
+[maven-lockfile plugin](https://github.com/chains-project/maven-lockfile). Maintainers
+are responsible for adding this plugin to their code and invoking the required Maven
+targets to generate the lockfile and build using a "frozen" `pom.xml` file.
+
```

Comment:
This sadly won't be enough for user facing docs, we'll need a more detailed `maven.md` docs describing the usage. Also, this is the only occurrence of `pom.xml` being mentioned. How does one generate it? Is Hermeto expected to do that? What information do we need to generate it in that case?

Reply:
> we'll need a more detailed maven.md docs describing the usage

Including an example walkthrough. PTAL at the .MDs for the individual package managers, paying special attention to the Example section (e.g. https://github.com/hermetoproject/hermeto/blob/main/docs/npm.md#example)